lib/travis/api/v3/queries/build.rb

```diff
@@ -12,11 +12,11 @@ def find
       raise WrongParams, 'missing build.id'.freeze
     end
 
-    def cancel(user)
+    def cancel(user, build_id)
```

Comment:
I am guessing, after making the code changes you have crossed checked that the existing cancel feature continues to work the same way 

the cancel of build is done three ways currently,

1. via web ui (https://blog.travis-ci.com/2013-08-21-abort-mission-cancel-running-builds)
2. via terminal command (https://blog.travis-ci.com/2013-08-21-abort-mission-cancel-running-builds)
3. via this way also https://docs.travis-ci.com/user/customizing-the-build/#building-only-the-latest-commit

Reply:
Yes, We have cross checked existing features(basically cancel the single build features). It will not effect existing features.